annotation-file-utilities/src/annotator/Source.java

```diff
@@ -67,9 +67,7 @@ public Source(String src) throws CompilerException, IOException {
         // Compiler options.
         // -Xlint:-options is a hack to get around Jenkins build problem:
         // "target value 1.8 is obsolete and will be removed in a future release"
-        final String[] stringOpts = new String[] { "-g", "-Xlint:-options" };
-            // "-XDTA:noannotationsincomments"
-          // TODO: figure out if these options are necessary? "-source", "1.6x"
+        final String[] stringOpts = new String[] { "-g", "-Xlint:-options", "-source", "8", "-target", "8" };
```

Comment:
The source/target 8 should only be added on a Java 8 JVM.
In CF we can use PluginUtils to determine the JVM version. Is there something similar in AFU?